api/services/DocumentService.ts

```diff
@@ -0,0 +1,80 @@
+import { ServiceOptions, Service } from 'ts-framework-common';
+import BitcapitalService from './BitcapitalService';
+import { DocumentType } from 'bitcapital-core-sdk';
+import {User} from '../models'
+import UserService from './UserService';
+
+
+export interface DocumentServiceOptions extends ServiceOptions{
+
+}
+
+export default class DocumentService extends Service{
+    protected static instance: DocumentService;
+
+    public async getDocumentStatusForUser(id: string){
+        const bitcapitalClient = BitcapitalService.getInstance({}).bitcapital;
+        const user: User = await UserService.getInstance({}).getUser(id);
+
+        return bitcapitalClient.consumers().findOne(user.bitcapitalId)
+                                .then(consumer => {return consumer.status});
+    }
+
+    public async setDocumentForUser(id: string, photoBase64: string){
+        const bitcapitalClient = BitcapitalService.getInstance({}).bitcapital;
+        const user: User = await UserService.getInstance({}).getUser(id);
+
+        await bitcapitalClient.consumers().createDocument(user.bitcapitalId, {
+            type: DocumentType.BRL_IDENTITY
+        });
+
+        const document = await bitcapitalClient.consumers().uploadDocumentPictureFromBase64(
+            user.bitcapitalId,
+            DocumentType.BRL_IDENTITY,
+            "front",
```

Comment:
Você só faz upload da parte da frente dos documentos?

Reply:
Sim...
Se realmente for importante faço a parte de trás, mas não sei...